js/DnD.js

```diff
@@ -0,0 +1,250 @@
+				var topDivTarget = getTopDiv(dragElement);
+				if(topDivTarget!=sevenStackArray[no]){
+					var tmpDivs = sevenStackArray[no].getElementsByTagName('DIV');					
+					var cardTypeThis = dragElement.id.substr(0,1);
+					var numericIDThis = dragElement.id.replace(/[^\d]/g,'');					
+					if(tmpDivs.length==0){
+						if(numericIDThis==13){
+							dragElement.style.top = '0px';
+							finishDrag(dragElement);
+							sevenStackArray[no].appendChild(dragElement);
+							
+							return dropped = true;
+						}						
+					}else{						
+						var destDiv = tmpDivs[tmpDivs.length-1];						
+						var cardTypeDest = destDiv.id.substr(0,1);						
+						var numericIDDest = destDiv.id.replace(/[^\d]/g,'');						
+						if(cardColors[cardTypeDest]!=cardColors[cardTypeThis] && numericIDDest-1==numericIDThis){
+							dragElement.style.top = verticalSpaceBetweenCards + 'px';
+							finishDrag(dragElement);
+							destDiv.appendChild(dragElement);
+							
+							return dropped = true; 
+						}
+					}
+				}	
+			}			
+		} return dropped = false;
+	}
+	
+	function appendToAces (dragElement, topPos, leftPos) {
+		for(var no=0;no<acesStackArray.length;no++){
+			
+			var tmpLeft = getleftPos(acesStackArray[no]);
+			var tmpTop = getTopPos(acesStackArray[no]);			
+			var tmpDivs = acesStackArray[no].getElementsByTagName('DIV');			
+			
+			if(leftPos>=tmpLeft-80 && leftPos<=tmpLeft+80 && topPos>=tmpTop-80 && topPos<=tmpTop+80){
+				var topDivTarget = getTopDiv(dragElement);
+				if(topDivTarget!=acesStackArray[no]){
+					var cardTypeThis = dragElement.id.substr(0,1);
+					var numericIDThis = dragElement.id.replace(/[^\d]/g,'');
+					if(tmpDivs.length==0){
+						if(numericIDThis==1){
+							dragElement.style.top = '0px';
+							finishDrag(dragElement);
+							acesStackArray[no].appendChild(dragElement);
+							return dropped = true;
+						}
+						
+					}else{
+						var destDiv = tmpDivs[tmpDivs.length-1];						
+						var cardTypeDest = destDiv.id.substr(0,1);						
+						var numericIDDest = destDiv.id.replace(/[^\d]/g,'');						
+						if(cardTypeDest==cardTypeThis && numericIDDest==(numericIDThis-1)){
+							dragElement.style.top = '0px';
+							finishDrag(dragElement);
+							destDiv.appendChild(dragElement);
+							dragElement.classList.remove('draggable');
+							if (checkWin()) {
+								alert("Congratulations! - you did it.");
+							} 
+						
+						return dropped = true;
+						}
+					}   
+				}	
+			}
+		}
+	}	
+	
+	function stopMoveCard(dragElement, dragZone) {
+		
+		var leftPos = parseInt(dragElement.style.left);
+		var topPos = parseInt(dragElement.style.top);	
+		
+		appendToSeven (dragElement, topPos, leftPos);
+		
+		if (dropped) {
+			return;
+		} else {
+			appendToAces (dragElement, topPos, leftPos);
+		}
+		
+		if (dropped) {
+			return;
+		} else {
+			rollback (dragElement, dragZone);
+			return;
+		}
+	}
+	// отменим действие по умолчанию на mousedown (выделение текста, оно лишнее)
+	return false;
+}	
```

Comment:
Unfortunately, this code is not yours. You got it with this link: http://creatorinfo.net/skript-pasyans-kosynka/. Everything you did is changing some functions names, I don't think it's enough to post this code as your own. And I have no reason to comment this code, sorry.